### DISCUSSION TITLE
docker reload param

### DISCUSSION THREAD

**Discussion by MrYZhou** [fastapi/fastapi#6282] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
```

### Description

docker --reload param is no work?

the file in docker container has been changed.but  the response is not new.

### Wanted Solution

need reload

### Wanted Code

```python
1
```

### Operating System

Linux

### FastAPI Version

0.79

### Python Version

3.6

**Comment by tiangolo** [comment]
I assume you solved it, thanks for closing the issue 👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
